src/main/java/org/broadinstitute/hellbender/utils/bigquery/BigQueryUtils.java

```diff
@@ -0,0 +1,449 @@
+package org.broadinstitute.hellbender.utils.bigquery;
+
+import com.google.cloud.bigquery.*;
+import com.google.cloud.bigquery.storage.v1beta1.*;
+import com.google.common.base.Preconditions;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericDatumReader;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.io.BinaryDecoder;
+import org.apache.avro.io.DatumReader;
+import org.apache.avro.io.DecoderFactory;
+import org.apache.ivy.util.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.StreamSupport;
+
+/**
+ * Utility class for dealing with BigQuery connections / tables / queries /etc.
```

Comment:
In the original Evoquer PR https://github.com/broadinstitute/gatk/pull/6011 there was a basic unit test for this branch:

https://github.com/broadinstitute/gatk/blob/111e6875cb29da5e7c75a5d113d217931ecd2c10/src/test/java/org/broadinstitute/hellbender/utils/bigquery/BigQueryUtilsUnitTest.java

We need some test coverage on this class, so we should either restore and expand `BigQueryUtilsUnitTest` or write a new set of tests from scratch. If we restore `BigQueryUtilsUnitTest` you should expand it slightly to cover additional overloads of `executeQuery()`.